spec/components/contents_list_spec.rb

```diff
@@ -77,6 +77,19 @@ def contents_list_with_markup
     assert_select ".gem-c-contents-list__link.govuk-link--no-underline[href='#two']", text: "2. Two"
   end
 
+  it "applies default margin to the component" do
+    render_component(contents: contents_list)
+    assert_select '.gem-c-contents-list.govuk-\!-margin-bottom-4'
+  end
+
+  it "applies margin to the component" do
```

Comment:
maybe make the description more explicit - "applies requested margin to the component" as it reads more like the default option as is.

Reply:
No problem, updated.